masim.c

```diff
@@ -492,7 +496,7 @@ int parse_phase(char *lines[], int nr_lines, struct phase *p,
 
 	p->name = (char *)malloc((strlen(lines[0]) + 1) * sizeof(char));
 	strcpy(p->name, lines[0]);
-	p->time_ms = atoi(lines[1]);
+	p->time_ms = atoll(lines[1]);
```

Comment:
The commit message was saying this change is for 64bit address, not time.  But I think this change is for supporting 64-bit time values?  Why we need this change?